hip/matrix/csr_kernels.hip.cpp

```diff
@@ -60,6 +60,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hip/base/types.hip.hpp"
 #include "hip/components/atomic.hip.hpp"
 #include "hip/components/cooperative_groups.hip.hpp"
+#include "hip/components/intrinsics.hip.hpp"
+#include "hip/components/merging.hip.hpp"
```

Comment:
Are you really using them here?

Reply:
Yes, see spgeam in common/matrix/csr_kernels.hpp.inc

Reply:
Oh, my bad, thanks for clearing that up.